ChangeLog.md

```diff
@@ -23,6 +23,19 @@ Current Trunk
 - dlopen, in conformace with the spec, now checks that one of either RTDL_LAZY
   or RTDL_NOW flags ar set.  Previously, it was possible set nether of these
   without generating an error.
+- Support added for INPUT_JS_VERSION and OUTPUT_JS_VERSION settings.  These
+  control is/when we use the closure compiler to transpile the resulting
```

Comment:
```suggestion
  control whether/when we use the closure compiler to transpile the resulting
```

Reply:
Oops that ChangeLog entry is actually from a different change .. fixed.